packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -51,6 +82,16 @@ const FeedbackStore = types
       }
     }
 
+    function showFeedback () {
+      self.showModal = true
+    }
+
+    function hideFeedback () {
+      self.showModal = false
+      self.allowSubjectAdvance = true
+      getRoot(self).subjects.advance()
```

Comment:
If we interrupt the advance action, then continue it again, we shouldn't need to couple feedback and subjects by calling `subjects.advance()` here. At the moment, the feedback modal will always advance the subject queue on close, no matter where it's used in the classifier.